packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -126,4 +126,7 @@ const ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): JS
   );
 };
 
+// TODO: Add treeshake wrapper
+const ActionList = React.memo(_ActionList);
```

Comment:
I'm a little skeptical about this. Can we very if dynamic updation of list items doesn't cause any issues?

Reply:
Sorry couldn't understand your concern here, can you expand on this?

Reply:
Yes I've done that.

Reply:
Toh why is there still a `// TODO: Add treeshake wrapper`?

Reply:
Oh wait, I think we don't need the treeshake wrapper here, because we are not assigning any static properties to the component eg: `displayName` or `componentId`

Reply:
Yep don't need it, rollup already adds the PURE comment: 

<img width="561" alt="image" src="https://user-images.githubusercontent.com/35374649/228439902-423ff025-f447-4b37-8ea8-9fafe3a1aa10.png">